node-operators/.nav.yml

```diff
@@ -3,5 +3,4 @@ hide: false
 nav:
   - index.md
   - networks
-  - oracle-nodes
```

Comment:
Niceeeee. When you create the PR for the redirects, can you also please delete the oracle icon used for the dropdown menu?